<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRCode</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lab_output">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Testo da convertire:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lE_testo"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pB_genera">
      <property name="text">
       <string>Genera QR</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="cB_svg">
      <property name="text">
       <string extracomment="Consente di salvare il qrcode in formato svg">svg file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pB_genera</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pB_valutaClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lE_testo</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>pB_valutaClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pB_valutaClick()</slot>
 </slots>
</ui>
